.github/workflows/docs.yml

```diff
@@ -19,10 +19,18 @@ jobs:
       with:
         python-version: 3.9
     - name: Upgrade pip and install wheel
-      run: pip install --upgrade pip wheel
+      run: pip install --upgrade pip wheel jupyterlab
     - name: Install pytket qsharp
       run: |        
         pip install .
+    - name: Install dotnet SDK
```

Comment:
adding here the same special handling of the iqsharp install for the doc build check